docs/ref/checks.txt

```diff
@@ -573,9 +573,6 @@ Templates
 The following checks verify that your :setting:`TEMPLATES` setting is correctly
 configured:
 
-* **templates.E001**: You have ``'APP_DIRS': True`` in your
```

Comment:
I think we should leave the docs in but add a note, like we already have for e.g. `security.W007`. An extra line like:

```
*This check is removed in Django 5.1 as system checks may now raise `ImproperlyConfigured` instead.*
```